.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseMergedDateTimeExtractor.cs

```diff
@@ -309,9 +314,10 @@ private List<ExtractResult> AddMod(List<ExtractResult> ers, string text)
                     success = TryMergeModifierToken(er, config.AfterRegex, text);
                 }
 
+                // The SinceRegex contains potential ambiguity part("from"), so the parameter potentialAmbiguity is true.
```

Comment:
We can re-phrase this to something like:
SinceRegex in English contains the term "from" which is potentially ambiguous with ranges in the form "from X to Y"
Also, please put the code comment before the method call, as it refers to it. Not before the 'if'